packages/gcf-utils/package.json

```diff
@@ -22,8 +22,9 @@
     "@google-cloud/kms": "^1.6.3",
     "@google-cloud/storage": "^4.0.0",
     "@octokit/plugin-enterprise-compatibility": "1.2.2",
+    "express": "^4.17.1",
```

Comment:
How did express slip in here?

Reply:
Not a slip, we were relying on `express` as a transitive dependency in `gcf-utils`. The new rules are, in my opinion, rightfully stricter about requiring that we actually define our dependencies.

There's risk involved in relying on a transitive dependency, as you don't have guarantees about what your top level library actually has access to.